DependencyInjection/SmartboxIntegrationFrameworkExtension.php

```diff
@@ -271,6 +273,49 @@ protected function loadQueueDrivers(ContainerBuilder $container)
                     }
                     break;
 
+                case 'phpamqplib':
+                    if (!class_exists(AMQPStreamConnection::class)) {
+                        throw new \RuntimeException('You need the amqp extension to use AMQP driver.');
```

Comment:
In this case, for the error message, we should mentioned that we are referring to phpAmqpLib

Reply:
Ok, done.